### DISCUSSION TITLE
GitHub Installation

### DISCUSSION THREAD

**Discussion by joydeep-git** [elementary/triage#179] [Q&A] [answered]
How to install the GitHub desktop? 
I'm a windows user, but going to try Elementary now. I Have found a way to install the apps I use the most instead of GitHub.

**Comment by RMcNeely** [comment, marked as answer]
Officially there is no Linux support (for any distro) for GitHub Desktop. You can probably get it running though if you follow the instructions for installing GitHub desktop on Ubuntu. There is also an **unofficial** copy that you can install from Flathub.
